gf-webapp/src/logic/rooms/roomPage/roomPromo/RoomPromo.module.scss

```diff
@@ -19,33 +19,24 @@
 }
 
 .title {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  position: absolute;
-  top: 0;
-  left: 0;
-  right: 0;
-  bottom: 0;
-  color: $color-white;
-  font-size: 60px;
-  font-family: Montserrat Regular;
+  @include room-h1;
 }
 
 .container,
 .services {
   background: $color-primary;
-  height: 50px;
+  min-height: 60px;
   display: flex;
   align-items: center;
-  justify-content: space-evenly;
+  height: fit-content;
 }
 
 .description {
   color: $color-text;
   text-align: center;
   font-weight: bold;
+  width: 50%;
+  margin: 5px 15px;
 }
```

Comment:
If possible move it to consts

Reply:
We don't have blocks with the same margin, so it has no sense.
We have 3 times property width: 50% in webapp:
```
  .image {
    width: 50%;
    margin: 0 auto;
    height: auto;
  }
```
```
  .roomAbout {
    width: 50%;
    padding: 30px;
  }
```
```
.description {
  color: $color-text;
  text-align: center;
  font-weight: bold;
  width: 50%;
  margin: 5px 15px;
}
```
How to unite them? I don't know)